src/main/java/org/vanilladb/core/query/planner/Verifier.java

```diff
@@ -210,10 +221,13 @@ public static void verifyCreateIndexData(CreateIndexData data,
 		Map<String, IndexInfo> indexInfoes = VanillaDb.catalogMgr().getIndexInfo(
 				tableName, tx);
 		if (indexInfoes.containsKey(fieldName))
-			throw new BadSemanticException("field" + fieldName
+			throw new BadSemanticException("field " + fieldName
 					+ " has already been indexed");
 	}
 
+	public static void verifyDropIndexData(DropIndexData data, Transaction tx) {
```

Comment:
We are planning to add `getIndex` by index names to `IndexMgr`. Hence, I hope you can add a comment `TODO: verify if the index exists` here as a placeholder for the future implementation.